packages/cubejs-schema-compiler/src/adapter/BaseQuery.js

```diff
@@ -2845,6 +2845,9 @@ export class BaseQuery {
       return evaluateSql === '*' ? '1' : evaluateSql;
     }
     if (this.ungrouped) {
+      if ((this.safeEvaluateSymbolContext().ungroupedAliases || {})[measurePath]) {
+        evaluateSql = (this.safeEvaluateSymbolContext().ungroupedAliases || {})[measurePath];
```

Comment:
If `IF` succeeded - no need to check again.
```suggestion
        evaluateSql = this.safeEvaluateSymbolContext().ungroupedAliases[measurePath];
```